### ISSUE TITLE
Add `--merge` flag to `k3sup plan` command

### ISSUE THREAD

**Issue by DaruZero** [alexellis/k3sup#438]
## Why do you need this?

I use the `plan` command to automate bootstrapping many clusters, and it would be useful to add `--merge` flag to the resulting script in order to avoid manually merging the kubeconfig.

## Expected Behaviour

Running the bootstrap script should require as little additional steps as possible.

## Current Behaviour

After bootstrapping a cluster, I need to manually merge the new kubeconfig with the default one

## Possible Solution

Add the `--merge` flag to the `plan` command, that would simply add the same `--merge` flag to the `install` command in the generated bootstrap script

## Your Environment

* k3sup version: 0.13.6

```
k3sup version
```

## Do you want to work on this?

Subject to design approval, are you willing to work on a Pull Request for this issue or feature request?

- [x] Yes
- [ ] No

**Comment by alexellis** [comment]
Hi @DaruZero please see the other comment I made here: https://github.com/alexellis/k3sup/issues/436#issuecomment-2273623928

If you'd like to send a pull request you can go ahead.

**Comment by DaruZero** [comment]
Yes, I'd like to work on this. I'll probably open a PR in a few days
